python/lsst/cp/pipe/utilsEfd.py

```diff
@@ -157,9 +158,9 @@ def selectTimeSeries(self, topicName, fields=[], startDate=None, endDate=None):
             List of fields to return.  If empty, all fields are
             returned.
         startDate : `astropy.time.Time`, optional
-            Start date (in UTC) to limit the results returned.
+            Start date (in TAI) to limit the results returned.
```

Comment:
Ok for here I don’t think this should specify. An astropy time can do conversions between tai and utc as needed, right? If it’s astropy it’s not a problem. Only strings become problems.

Reply:
Right. Use startDate.tai